src/content/content.js

```diff
@@ -31,27 +41,41 @@ if (!amLoaded) {
 						"saveAs"
 					]
 				}
-				if (document.documentElement.requestFullScreen || document.documentElement.mozRequestFullScreen) {
-					//result.enable.push("fullscreen");
-				}
-				if (document.querySelector(":focus")) {
+
+				if (isFocusable && focusedElement.isContentEditable) {
 					result.enable.push("edit*");
+				} else if (isFocusable) {
+					result.enable.push("editCut", "editCopy", "editSelectAll");
+				} else {
+					result.enable.push("editCopy", "editSelectAll");
```

Comment:
Copy is still broken in this context on Windows. That’s why I implemented [commit `ad96f85`](https://github.com/ExE-Boss/app-menu/commit/ad96f85bac7397d4b6afab32c1ec78d78b54a4bd), so that this can be disabled on Windows and enabled on MacOS.